<?xml version="1.0" ?>
<launch>

  <arg name="topics" default=""/>
  <arg name="config_file" default=""/>
  <arg name="auto_safety_tagging" default="true"/>
  <arg name="safety_pub_rate" default="10.0"/>
  <arg name="map_pub_rate" default="0"/>
  <arg name="map_plt_rate" default="0"/>

  <node pkg="sentor" type="sentor_node.py" name="sentor" output="screen">
    <param name="~config_file" value="$(arg config_file)" />
    <param name="~auto_safety_tagging" value="$(arg auto_safety_tagging)" />
    <param name="~safety_pub_rate" value="$(arg safety_pub_rate)" />
    <param name="~map_pub_rate" value="$(arg map_pub_rate)" />
    <param name="~map_plt_rate" value="$(arg map_plt_rate)" />
  </node>	

</launch>
